ansible/roles-infra/infra-osp-template-create/tasks/create_osp_template.yml

```diff
@@ -1,69 +1,100 @@
+- name: Check for running Heat stacks
+  command: >-
+    openstack stack list -f json
```

Comment:
Are we sure this will work and that `openstack stack list` won't page results? Like first 50 of 200?

Should we perhaps filter the list by creating as @rut31337 suggested?

Reply:
I have not seen this, but I am not sure. We need the full list since we are removing the ability to update a stack...I need the full list to determine whether the stack exists or not and whether we bail or proceed with creating a new one. Only pulling back a filtered list would break this.